lib/core/viewmodels/security_viewmodel.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'package:notredame/core/constants/emergency_procedures.dart';
+import 'package:notredame/core/constants/markers.dart';
+import 'package:notredame/core/models/emergency_procedure_model.dart';
+import 'package:stacked/stacked.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class SecurityViewModel extends BaseViewModel {
+  List<Marker> markersList = markers;
+
+  List<EmergencyProcedure> emergencyProcedureList = emergencyProcedures;
+
+  Set<Marker> getMarkers(List<Marker> markersList) {
+    final Map<MarkerId, Marker> markers = <MarkerId, Marker>{};
+
+    for (int i = 0; i < markersList.length; i++) {
+      markers[markersList[i].markerId] = markersList[i];
+    }
+    return Set<Marker>.of(markers.values);
+  }
+
+  Future<void> makePhoneCall(String url) async {
```

Comment:
The function doesn't 'make a phone call' but redirect to phone application of the phone. If you can also add some documentation that would be perfect 😄

Reply:
done